tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
I guarantee there's some coding standard applied to those tests also. Stylecop or not they need to be cleaner, conform to a basic set of rules and naming standards and be instantly obvious what they are testing.